### PR TITLE
maintainers: Update maintainer for sgi/sgm platforms

### PR DESCRIPTION
**Pull request by npoushin**

### CODE REVIEW

File: maintainers.rst [hunk 1]
```diff
@@ -30,6 +30,11 @@ Armv7-A architecture port
 :M: Etienne Carriere <etienne.carriere@linaro.org>
 :G: `etienne-lms`_
 
+Arm System Guidance for Infrastructure / Mobile FVP platforms
+-------------------------------------------------------------
+:M: Nariman Poushin <nariman.poushin@linaro.org>
+:G: `nariman`_
```
Comment (antonio-nino-diaz-arm) [inline maintainers.rst]: Can you add the paths to the folders that you are going to be the maintainer of?

**Comment by antonio-nino-diaz-arm** [comment]
IP Review +1

File: maintainers.rst (hunk 1 above)
Comment (npoushin) [reply maintainers.rst]: Done

Comment (antonio-nino-diaz-arm) [reply maintainers.rst]: What about ``plat/arm/board/{sgi575,sgm775}``?

Comment (npoushin) [reply maintainers.rst]: Of course, apologies for omitting. Fixed.

**Pull request merged**